diff --git a/Modules/timemodule.c b/Modules/timemodule.c
index 1bfbf3f6a0b..ff006ea969c 100644
--- a/Modules/timemodule.c
+++ b/Modules/timemodule.c
@@ -41,7 +41,8 @@
 #  include <sanitizer/msan_interface.h>
 #endif

-#ifdef _MSC_VER
+//#ifdef _MSC_VER
+#if 1
 #  define _Py_timezone _timezone
 #  define _Py_daylight _daylight
 #  define _Py_tzname _tzname
@@ -187,7 +188,7 @@ time_clockid_converter(PyObject *obj, clockid_t *p)
 {
 #ifdef _AIX
     long long clk_id = PyLong_AsLongLong(obj);
-#elif defined(__DragonFly__)
+#elif 1
     long clk_id = PyLong_AsLong(obj);
 #else
     int clk_id = PyLong_AsInt(obj);
diff --git a/Python/fileutils.c b/Python/fileutils.c
index 78603d40704..c6d9b50cd03 100644
--- a/Python/fileutils.c
+++ b/Python/fileutils.c
@@ -47,7 +47,7 @@ extern int winerror_to_errno(int);

    The flag is used by _Py_open(), _Py_open_noraise(), io.FileIO
    and os.open(). */
-int _Py_open_cloexec_works = -1;
+int _Py_open_cloexec_works = 1;
 #endif

 // The value must be the same in unicodeobject.c.
@@ -1830,7 +1830,7 @@ Py_fopen(PyObject *path, const char *mode)
         return NULL;
     }

-    if (set_inheritable(fileno(f), 0, 1, NULL) < 0) {
+    if (0 && set_inheritable(fileno(f), 0, 1, NULL) < 0) {
         fclose(f);
         return NULL;
     }
@@ -3045,7 +3045,8 @@ _Py_closerange(int first, int last)
 int
 _Py_GetTicksPerSecond(long *ticks_per_second)
 {
-#if defined(HAVE_SYSCONF) && defined(_SC_CLK_TCK)
+//#if defined(HAVE_SYSCONF) && defined(_SC_CLK_TCK)
+#if 0
     long value = sysconf(_SC_CLK_TCK);
     if (value < 1) {
         return -1;
diff --git a/Python/thread_pthread.h b/Python/thread_pthread.h
index da405824244..a6c52e38458 100644
--- a/Python/thread_pthread.h
+++ b/Python/thread_pthread.h
@@ -83,7 +83,7 @@
 /* The POSIX spec says that implementations supporting the sem_*
    family of functions must indicate this by defining
    _POSIX_SEMAPHORES. */
-#ifdef _POSIX_SEMAPHORES
+#if 0
 /* On FreeBSD 4.x, _POSIX_SEMAPHORES is defined empty, so
    we need to add 0 to make it work there as well. */
 #if (_POSIX_SEMAPHORES+0) == -1
